src/adapters/refract/transformSections.coffee

```diff
@@ -39,16 +39,18 @@ module.exports = (parentElement) ->
 
       description = getDescription(element)
 
-      # Then create a new section in the Application AST
-      # corresponding to the Category element.
-      resourceGroup = new blueprintApi.Section({
-        name: _.chain(element).get('meta.title', '').contentOrValue().value()
-        description: description.raw
-        htmlDescription: description.html
-        resources: transformResources(element)
-      })
+      classes = _.get(element, 'meta.classes', [])
```

Comment:
Just as a note, I think you can do some cool stuff with `matches`.

```js
lodash = require("lodash")

check = lodash.matches({meta:{classes:['resourceGroup']}});

// false
fail = check({
    element: 'foo',
    content: 'bar'
});

// true
pass = check({
    element: 'foo',
    meta: {
        classes: ['resourceGroup', 'another']
    },
    content: 'bar'
});
```

Just an FYI.